api/envoy/service/cluster/v2alpha/outlier_detection_event_service.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
```

Comment:
Should we have a single Envoy event streaming service? It seems less boiler plate and implementation work.

Reply:
+1, the events service could take a repeated oneof of events, or even could take typed Any objects?

As long as we allow each event source to specify an independent service it would remain flexible on how the deployment implements it.

Reply:
Yeah, I wonder if we want a model like ADS where it's possible to have multiple event sources share a service. I'm thinking of a use case like Envoy anomaly detection, where everything gets fed in-sequence into a giant ML soup.

Reply:
Yes exactly.

Reply:
aggregated service semms good, but I wonder what would be the best way to configure sucn endpoint. Should it be some sort of a global bootstrap-level endpoint, or per cluster?

Reply:
I would still make it per source like you have now and just use a common service definition. IMO in the future we could add aggregation support?

Reply:
Yep, this is what I was thinking it would look like. My other thought is that in V4 we could fold metrics service sink and the grpc log sink into the "event sink" service definition as well.

Reply:
I've changed this to a single AggregatedEventService, PTAL